### PR TITLE
feat: add editor texture and camera manual

### PR DESCRIPTION
**Pull request by GuoLei1990**
Add editor texture and camera manual.

### CODE REVIEW

File: docs/editor-camera.zh-CN.md [hunk 1]
```diff
@@ -4,19 +4,28 @@ title: 相机组件
...
-![image.png](https://gw.alipayobjects.com/mdn/rms_d27172/afts/img/A*9WfrRp6fLIcAAAAAAAAAAAAAARQnAQ)
+2. 选中相机即可。
 
+<img src="/Users/guolei/Library/Application Support/typora-user-images/image-20210720164414642.png" alt="image-20210720164414642" style="zoom:50%;" />
 
-3. 然后在编辑器里就可以很方便的修改相机属性了
+### 属性面板
 
-![image.png](https://gw.alipayobjects.com/mdn/rms_d27172/afts/img/A*RTmNSL8iDzcAAAAAAAAAAAAAARQnAQ)
+点击 Camera 所在的实体**检查面板**会显示 Camera 组件的相关属性。
```
Comment (czizzy) [inline docs/editor-camera.zh-CN.md:23]: camera相关属性介绍一下吧，参考play canvas: https://developer.playcanvas.com/en/user-manual/packs/components/camera/，或者引擎有对应的文档链接过去也行

File: docs/editor-texture.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+在三维对象中使用纹理，需要将纹理应用于 [Material](${docs}material-cn) 对象，再将 Material 对象赋值给 [MeshRenderer](${docs}mesh-renderer-cn) 和 [ParticleRenderer](${docs}particle-renderer-cn) 等三维对象渲染器。一个 Material 对象可以包含多张 Texture，多个 Material 也可以共享一张 Texture。
+
+![TextureToMaterial](https://gw.alipayobjects.com/zos/OasisHub/ba930f61-6ec7-49e8-814b-b6c24a078f0f/TextureToMaterial.gif)
+
+#### 二维对象 
+
+在二维对象对象中使用纹理，需要将纹理应用于 [Sprite](${docs}sprite-cn) 对象，再将 Sprite 对象赋值给 [SpriterRenderer](${docs}sprite-renderer-cn) 。多个 Sprite 对象可以共享同一张 Texture。
+
+![TextureToSprite](https://gw.alipayobjects.com/zos/OasisHub/16109017-fe5e-45f8-bd66-51abfcedf60e/TextureToSprite.gif)
```
Comment (czizzy) [inline docs/editor-texture.zh-CN.md:38]: 和上面一样，texture的property也介绍一下

**Review by czizzy — CHANGES_REQUESTED** [review]

File: docs/editor-texture.zh-CN.md (hunk 2 above)
Comment (czizzy) [reply docs/editor-texture.zh-CN.md:38]: 还有纹理的压缩功能也要介绍

File: docs/editor-camera.zh-CN.md (hunk 1 above)
Comment (GuoLei1990) [reply docs/editor-camera.zh-CN.md:23]: done

**Review by czizzy — APPROVED** [review]
+1

**Review by zhang2333 — APPROVED** [review]

**Pull request merged**